terraform/outputs.tf

```diff
@@ -19,6 +19,6 @@ output "cluster_name" {
 }
 
 output "primary_zone" {
```

Comment:
Maybe worth changing this output to primary_location now?

Reply:
It's done.